app/services/processing_service.py

```
@@ -613,50 +937,130 @@ async def process_repository(
-            # -----------------------
-            # ANALYSIS
-            # -----------------------
-            _ = await self.analyze_repository(
-                chunks,
-                relative_path,
-                repo.language,
-                repo.id,
-                job_tracker_instance=job_tracker_instance,
-            )
+            for package_info in files:
+                for package_name, package_item in package_info.items():
```

Ruff (0.14.4)

Loop control variable `package_name` not used within loop body

Rename unused `package_name` to `_package_name`

(B007)